model-optimizer/extensions/ops/sparse_reshape.py

```diff
@@ -3,13 +3,14 @@
 
 import numpy as np
 
+from mo.front.common.partial_infer.utils import dynamic_dimension, is_fully_defined, shape_array, strict_compare_tensors
 from mo.graph.graph import Node, Graph
 from mo.ops.op import Op
 
 
 class SparseReshape(Op):
     """
-    SparseReshape operation reshapes a sparse tensor. It recomputes indices for a new dense shape.
+    SparseReshape operation reshapes a sparse tensor in COO format. It recomputes indices for a new dense shape.
```

Comment:
what is COO format? please explain this abbreviation.

Reply:
it's storing sparse matrices in https://en.wikipedia.org/wiki/Sparse_matrix#Coordinate_list_(COO) format. I will concretize this in comments.

SparseReshape operation reshapes a sparse tensor in Coordinate list (COO) format. 

Does this sound ok?

Reply:
added concretization